### PR TITLE
DOC: improve the examples in README

### PR DESCRIPTION
**Pull request by lryong**

### CODE REVIEW

File: smtp.go [hunk 1]
```diff
@@ -20,6 +20,8 @@ type SMTP struct {
 }
 
 // CheckSMTP performs an email verification on the passed domain via SMTP
+//  - domain is the passed email domain
```
Comment (git-hulk) [inline smtp.go:23]:
```suggestion
//  - the domain is the passed email domain
```

File: smtp.go [hunk 2]
```diff
@@ -20,6 +20,8 @@ type SMTP struct {
...
+//  - username is used to check the deliverability of specific email address, if server is catch-all server, username will not be checked
```
Comment (git-hulk) [inline smtp.go:24]: the username is used to check the deliverability of a specific email address, if the server is a catch-all server, the username will not be checked

Comment (git-hulk) [reply smtp.go:24]: split the comment into multi-lines?

**Review by git-hulk — APPROVED** [review]

File: smtp.go (hunk 2 above)
Comment (lryong) [reply smtp.go:24]: Yes, that's better.

**Review by chuan-T — APPROVED** [review]

**Review by git-hulk — APPROVED** [review]

**Pull request merged**